### PR TITLE
Make sure docker is only checked on non-windows gradle check

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
Make sure docker is only checked on non-windows gradle check

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/3816

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/gradle/gradle-check.jenkinsfile [hunk 1]
```diff
@@ -100,35 +97,39 @@ pipeline {
...
-                        if (!env.BUILD_CAUSE.contains('Started by user') && !env.BUILD_CAUSE.contains('Started by timer')) {
-                            def pr_url = "${pr_to_clone_url}".replace(".git", "/pull/${pr_number}")
-                            println("Triggered by GitHub: ${pr_to_clone_url}")
-                            if ("$pr_number" == "Null") {
-                                currentBuild.description = """runner: ${agent_name}<br><a href="${pr_to_clone_url}">Others</a>: ${pr_title}"""
-                            }
-                            else {
-                                currentBuild.description = """runner: ${agent_name}<br><a href="${pr_url}">PR #${pr_number}</a>: ${pr_title}"""
-                            }
+                    if ( ! params.AGENT_LABEL.toLowerCase().contains('windows')) {
```
Comment (rishabh6788) [inline jenkins/gradle/gradle-check.jenkinsfile:100]: Question: What happens when the platform is windows?

File: jenkins/gradle/gradle-check.jenkinsfile [hunk 2]
```diff
@@ -100,35 +97,39 @@ pipeline {
...
+                    }
 
-                            runGradleCheck(
-                                gitRepoUrl: "${pr_from_clone_url}",
-                                gitReference: "${pr_from_sha}"
-                            )
+                    if (!env.BUILD_CAUSE.contains('Started by user') && !env.BUILD_CAUSE.contains('Started by timer')) {
+                        def pr_url = "${pr_to_clone_url}".replace(".git", "/pull/${pr_number}")
+                        println("Triggered by GitHub: ${pr_to_clone_url}")
+                        if ("$pr_number" == "Null") {
```
Comment (rishabh6788) [inline jenkins/gradle/gradle-check.jenkinsfile:109]: Didn't understand this case?
Can a PR not have a PR number or is this some other trigger?

File: jenkins/gradle/gradle-check.jenkinsfile (hunk 1 above)
Comment (peterzhuamazon) [reply jenkins/gradle/gradle-check.jenkinsfile:100]: As of now windows does not have docker equipped.
However, I have recently add some new development in https://github.com/opensearch-project/opensearch-ci/issues/281 so this PR might not need any changes anymore.

File: jenkins/gradle/gradle-check.jenkinsfile (hunk 2 above)
Comment (peterzhuamazon) [reply jenkins/gradle/gradle-check.jenkinsfile:109]: This is just changed due to spacing, the old itself is not changed for months.
It literally just to determine whether if this is triggered by PR then assign number of that PR in description, else dont.

**Comment by peterzhuamazon** [comment]
We can close this now as I have new solutions since we have implemented Windows containers now.

**Pull request closed without merging**